integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonTestUtils.java

```diff
@@ -1305,6 +1306,55 @@ public static void createDomainAndVerify(Domain domain, String domainNamespace)
     return ingressHostList;
   }
 
+
+  /**
+   * Execute command inside a pod and assert the execution.
+   *
+   * @param pod V1Pod object
+   * @param containerName name of the container inside the pod
+   * @param redirectToStdout if true redirect to stdout and stderr
+   * @param command the command to execute inside the pod
+   */
+  public static void execInPod(V1Pod pod, String containerName, boolean redirectToStdout, String command) {
+    LoggingFacade logger = getLogger();
+    ExecResult exec = null;
+    try {
+      logger.info("Executing command {0}", command);
+      exec = Exec.exec(pod, containerName, redirectToStdout, "/bin/sh", "-c", command);
+      // checking for exitValue 0 for success fails sometimes as k8s exec api returns non-zero
+      // exit value even on success, so checking for exitValue non-zero and stderr not empty for failure,
+      // otherwise its success
+      assertFalse(exec.exitValue() != 0 && exec.stderr() != null && !exec.stderr().isEmpty(),
+          String.format("Command %s failed with exit value %s, stderr %s, stdout %s",
+              command, exec.exitValue(), exec.stderr(), exec.stdout()));
+    } catch (IOException | ApiException | InterruptedException ex) {
+      logger.warning(ex.getMessage());
+    }
+  }
+
+
+  /**
+   * Copy a file to the pod.
+   *
+   * @param namespace the namespace in which the pod exists
+   * @param podName name of the pod
+   * @param containerName name of the container inside the pod
+   * @param srcPath source path of the file to copy
+   * @param dstPath destination path for the file inside the pod
+   */
+  public static void copyFileToPod(String namespace, String podName,
+      String containerName, Path srcPath, Path dstPath) {
+    LoggingFacade logger = getLogger();
+    try {
+      logger.info("Copying file {0} inside pod location {1}", srcPath, dstPath);
+      Kubernetes.copyFileToPod(namespace, podName, containerName, srcPath, dstPath);
+    } catch (ApiException | IOException ex) {
+      logger.warning(ex.getMessage());
+    }
+  }
+
+
+
```

Comment:
You can use existing FileUtils.copyFileToPod(String namespace,
                                   String pod,
                                   String container,
                                   Path srcPath,
                                   Path destPath)

Reply:
Using  it from FileUtils